provers/sp1/driver/src/lib.rs

```diff
@@ -122,9 +121,7 @@ impl Prover for Sp1Prover {
             let proof_id = network_prover
                 .request_proof(ELF, stdin, param.recursion.clone().into())
                 .await
-                .map_err(|e| {
-                    ProverError::GuestError(format!("Sp1: requesting proof failed: {e}"))
-                })?;
+                .map_err(|_| ProverError::GuestError("Sp1: requesting proof failed".to_owned()))?;
```

Comment:
More detailed error 'e' not available anymore?

Reply:
wdym not available anymore? this code change does the exact same thing as before and it's merged from #349 and you approved it before

Reply:
It doesn't print `e` anymore in the error message so less detailed message now?

Reply:
yes, should have error detail showed here.